### DISCUSSION TITLE
jsonschema 中的验证 uniqueItems 问题

### DISCUSSION THREAD

**Discussion by yaoleia** [alibaba/formily#3363] [Q&A 2.x]
uniqueItems: true  关键字用于约束数组所有元素值是唯一的。

目前的逻辑是不是约束所有元素值都是相等的，是不是判断反了？

https://github.com/alibaba/formily/blob/72534b43284921574631edb514e591a3bbf950dd/packages/validator/src/rules.ts#L129-L142

**Comment by janryWang** [comment]
语义的确有点问题，可以来个PR修复一下
